lib/install/config/webpacker.yml

```diff
@@ -4,7 +4,8 @@ default: &default
   source_path: app/javascript
   source_entry_path: packs
   public_output_path: packs
-  compile_missing_packs: true
+
+  compile_missing_packs: false
```

Comment:
I wasn't quite clear on the usage and it seemed that the main purpose of this is to compile assets (any desired env) during lookup when running tests. For rest of the cases, either dev server or watcher can be used. By default we turn if off and can be toggled per env. WDYT?